app/src/main/java/com/internship/move/utils/extensions/CustomDialogFragment.kt

```diff
@@ -23,9 +23,9 @@ class CustomDialogFragment : DialogFragment(R.layout.item_custom_dialog) {
     private fun initDialog() {
```

Comment:
Regarding the `CustomDialogFragment` name, I suggest renaming it to something more meaningful. This class is indeed "custom" but that doesn't tell us what its job is. Avoid using `Custom` in names because at the end of the day everywthing we do is "custom" in some sense :D